dts/bindings/can/can-controller.yaml

```diff
@@ -30,3 +30,42 @@ properties:
         Sample point in permille.
         This param is required if segments are not given.
         If the sample point is given, the segments are ignored.
+    phys:
+      type: phandle
+      required: false
+      description: |
+        Actively controlled CAN transceiver.
+
+        Example:
+          tja1040: can-transceiver {
+            compatible = "nxp,tja1040", "can-transceiver-gpio";
+            standby-gpios = <gpioa 0 GPIO_ACTIVE_HIGH>;
+            max-bitrate = <1000000>;
+            #phy-cells = <0>;
+          };
+
+          &can0 {
+            status = "okay";
+
+            phys = <&tja1040>;
+          };
```

Comment:
Should be updated to match the code change as discussed before.

Reply:
Thanks, fixed.